docs/index.mdx

```diff
@@ -19,13 +19,13 @@ The [API Reference](https://aserto.readme.io/reference) contains (executable) do
 
 The Aserto documentation is organized as follows:
 
-- [Overview](/docs/overview/introduction): a conceptual overview of Aserto's different moving parts
-- [Getting Started](/docs/getting-started-acmecorp/quickstart): a step-by-step walkthrough to get a demo app up-and running using Aserto
-- [Aserto Console](/docs/aserto-console/introduction): an overview of the workflows supported by the Aserto Console
-- [Authorizer Guide](/docs/authorizer-guide/overview): a guide for interacting with the Aserto Authorizer
-- [Quickstarts](/docs/quickstarts/react/overview): a guide for creating a React application and an Express.js service that utilize Aserto for authorization
-- [SDKs](/docs/software-development-kits/overview): the Aserto language-specific SDKs
-- [Command-Line Interfaces](/docs/command-line-interface/introduction): an overview of the Aserto and Policy CLIs
-- [Decisions Logs Guide](/docs/decision-logs-guide/overview): a guide for using Aserto decision logs
-- [Edge Authorizers](/docs/edge-authorizers/overview): a guide for setting up edge authorizers
-- [Troubleshooting](/docs/troubleshooting/overview): a list of common problems and how to resolve them
+- [Overview](docs/overview/introduction): a conceptual overview of Aserto's different moving parts
```

Comment:
I found that putting the leading slash (`/docs/...`) helps make sure that no matter where the page is in the hierarchy, it will resolve correctly. previously, these all broke when I did the "trailingSlash: false" change